repo2docker/buildpacks/_r_base.py

```diff
@@ -55,12 +58,8 @@ def rstudio_base_scripts():
             "${NB_USER}",
             # Install nbrsessionproxy
             r"""
-                pip install --no-cache-dir jupyter-server-proxy==1.4.0 && \
-                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-rsession-proxy/archive/d5efed5455870556fc414f30871d0feca675a4b4.zip && \
-                pip install --no-cache-dir https://github.com/ryanlovett/jupyter-shiny-proxy/archive/47557dc47e2aeeab490eb5f3eeae414cdde4a6a9.zip && \
-                jupyter serverextension enable jupyter_server_proxy --sys-prefix && \
-                jupyter nbextension install --py jupyter_server_proxy --sys-prefix && \
-                jupyter nbextension enable --py jupyter_server_proxy --sys-prefix
+                pip install --no-cache-dir jupyter-rsession-proxy==1.4 && \
+                pip install --no-cache-dir https://github.com/ryanlovett/jupyter-shiny-proxy/archive/47557dc47e2aeeab490eb5f3eeae414cdde4a6a9.zip
```

Comment:
Is this ready to merge, or do you want to wait for a release?

Reply:
I think this is ready to merge now - I just haven't touched the shiny-proxy setup for now.

Reply:
Hmm actually - maybe let's wait a few days, and see if we can get up to a much more modern rstudio.

Reply:
@manics ok I've upgraded this now to use latest rstudio and jupyter-rsession-proxy.